packages/html-plugin/index.js

```diff
@@ -70,12 +70,20 @@ const defaultTemplate = ({
 <html>
   <head>
     <meta charset='utf-8'>
-    <meta name='viewport' content='width=device-width,initial-scale=1'>
+    <meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0'/>
     <style>*{box-sizing:border-box}body{margin:0;font-family:system-ui,sans-serif}</style>
     <meta name='generator' content='mdx-deck'>
+    <link rel="manifest" href="/manifest.json">
```

Comment:
The PWA setup isn't necessary for the touchscreen/mobile UI changes, is it? I'd rather not introduce this here since I have other plans related to this